iset.mm

```diff
@@ -125588,6 +125608,328 @@ reduced fraction representation (no common factors, denominator
+    elcncf1i.3 $e |- ( ( ( x e. A /\ w e. A ) /\ y e. RR+ ) ->
+                       ( ( abs ` ( x - w ) ) < Z ->
+                         ( abs ` ( ( F ` x ) - ( F ` w ) ) ) < y ) ) $.
+    $( Membership in the set of continuous complex functions from ` A ` to
+       ` B ` .  (Contributed by Paul Chapman, 26-Nov-2007.) $)
+    elcncf1ii $p |- ( ( A C_ CC /\ B C_ CC ) -> F e. ( A -cn-> B ) ) $=
+      ( cc wss wa co wcel wi wtru a1i cv cfv wf crp cmin cabs clt wbr elcncf1di
+      ccncf mptru ) DKLEKLMFDEUHNOPQABCDEFGDEFUAQHRASZDOZBSZUBOZMGUBOPQIRUKCSZD
+      OMUMMUJUNUCNUDTGUEUFUJFTUNFTUCNUDTULUEUFPPQJRUGUI $.
+  $}
+
+  ${
+    $d f w x y z A $.  $d f w x y z B $.  $d f w x y z C $.  $d w x y z F $.
+    $( A continuous complex function restricted to a subset is continuous.
+       (Contributed by Paul Chapman, 18-Oct-2007.)  (Revised by Mario Carneiro,
+       25-Aug-2014.) $)
+    rescncf $p |- ( C C_ A ->
+            ( F e. ( A -cn-> B ) -> ( F |` C ) e. ( C -cn-> B ) ) ) $=
+      ( vx vw vz vy wss co wcel wa cv cmin cabs cfv clt wral crp cc cres wf wbr
+      ccncf wi wrex simpr wb cncfrss adantl cncfrss2 elcncf syl2anc mpbid simpl
+      simpld fssresd simprd ssralv fvres oveqan12d fveq2d breq1d imbi2d biimprd
+      ralimdva sylan9 reximdv ralimdv syld sylc sstrd mpbir2and ex ) CAIZDABUDJ
+      KZDCUAZCBUDJKZVOVPLZVRCBVQUBZEMZFMZNJOPGMQUCZWAVQPZWBVQPZNJZOPZHMZQUCZUEZ
+      FCRZGSUFZHSRZECRZVSABCDVSABDUBZWCWADPZWBDPZNJZOPZWHQUCZUEZFARZGSUFZHSRZEA
+      RZVSVPWOXELZVOVPUGVSATIZBTIZVPXFUHVPXGVOABDUIUJZVPXHVOABDUKUJZEHGFABDULUM
+      UNZUPVOVPUOZUQVSVOXEWNXLVSWOXEXKURVOXEXDECRWNXDECAUSVOXDWMECVOWACKZLZXCWL
+      HSXNXBWKGSVOXBXAFCRXMWKXAFCAUSXMXAWJFCXMWBCKZLZWJXAXPWIWTWCXPWGWSWHQXPWFW
+      ROXMXOWDWPWEWQNWACDUTWBCDUTVAVBVCVDVEVFVGVHVIVFVJVKVSCTIXHVRVTWNLUHVSCATX
+      LXIVLXJEHGFCBVQULUMVMVN $.
+
+    $( Change the codomain of a continuous complex function.  (Contributed by
+       Paul Chapman, 18-Oct-2007.)  (Revised by Mario Carneiro, 1-May-2015.) $)
+    cncffvrn $p |- ( ( C C_ CC /\ F e. ( A -cn-> B ) ) ->
+      ( F e. ( A -cn-> C ) <-> F : A --> C ) ) $=
+      ( vw vx vz vy cc wss ccncf co wcel wa cv cmin cabs cfv wral crp wf clt wi
+      wbr wrex wb cncfrss simpl elcncf2 syl2anc cncfi 3expb ralrimivva biantrud
+      adantl bitr4d ) CIJZDABKLMZNZDACKLMZACDUAZEOZFOZPLQRGOUBUDVBDRVCDRPLQRHOZ
+      UBUDUCEASGTUEZHTSFASZNZVAUSAIJZUQUTVGUFURVHUQABDUGUOUQURUHFHGEACDUIUJUSVF
+      VAURVFUQURVEFHATURVCAMVDTMVEGEABVCVDDUKULUMUOUNUP $.
+
+    $( The set of continuous functions is expanded when the range is expanded.
+       (Contributed by Mario Carneiro, 30-Aug-2014.) $)
+    cncfss $p |- ( ( B C_ C /\ C C_ CC ) -> ( A -cn-> B ) C_ ( A -cn-> C ) ) $=
+      ( vf wss cc wa ccncf co cv wcel cncff adantl simpll fssd cncffvrn adantll
+      wf wb mpbird ex ssrdv ) BCEZCFEZGZDABHIZACHIZUEDJZUFKZUHUGKZUEUIGZUJACUHR
+      ZUKABCUHUIABUHRUEABUHLMUCUDUINOUDUIUJULSUCABCUHPQTUAUB $.
```

Comment:
Probably shorter to prove cncfss first and pretty directly from the definition of -cn->

Then, a probably more useful version of cncffvrn is
```
$p |- ( ( C C_ CC /\ ( F e. ( A -cn-> B ) /\ ( F " A ) C_ C ) ) -> F e. ( A -cn-> C ) ) $=
```

More generally, I'm not convinced by this -cn-> definition. It would probably be more useful to define -Top-> such that `( A -Top-> B )` is the set of continuous maps from the topological space A to the topological space B, in the frameworl of extensible structures. This would avoid reproving general truths about continuous functions. I know it's not about iset.mm, so feel free to open an issue, or I'll open one myself if I find the motivation for it.

Reply:
> I know it's not about iset.mm

Yeah, I'm not qualified to address pretty much all these comments. I really don't know this section of set.mm at all. Sometimes I learn a bit when I intuitionize a section but even that can be a very mechanical process which doesn't require a lot of engaging with deeper mathematical issues until/unless the proofs hit something which uses excluded middle in a way which is not easily replaced.

Sure, this is partly an issue of what I am or am not interested in, but it also is a pretty large amount of material which I don't know well enough to even have an ability to say what kinds of rearrangements would be good or bad.